.ddev/config.yaml

```diff
@@ -31,8 +31,8 @@ hooks:
     # Import any local config split profile overrides.
     - exec: "drush cim -y"
 name: dept
-nodejs_version: 20
-php_version: "8.2"
+nodejs_version: 14
```

Comment:
Will this break the npm fixes Johan did last week?

Reply:
Not sure. I went with the most recent looking changes (the PHP version update)